test/functional/test_site/testNunjuckUtils.md

```diff
@@ -0,0 +1,9 @@
+{%raw%}
+
+The standard trick to render braces still works {<span>{ content }</span>}.
+
+Nonexistent variables should still be removed: {{ unknown_variable }}. In the rare event that user wrapped an unknown variable inside a raw tag, it should retain its current behaviour and not be produced to ensure Vue does not break.
+
+Random braces should be removed: { }
```

Comment:
Isn't this due to `markdown-it-attrs`?

Reply:
e.g. something like 
```
<div>
lorem { ipsum }
</div>
```
should still have the braces afterward

Reply:
Removed this test, as this is not really related to the PR